packages/caip/src/accountId/accountId.ts

```diff
@@ -42,24 +57,25 @@ export const fromAccountId: FromAccountId = (accountId) => {
     throw new Error(`fromAccountId: invalid AccountId ${accountId}`)
   }
 
+  const chainNamespace = parts[0]
+  const chainReference = parts[1]
```

Comment:
Given an account id `cosmos:cosmoshub-4/slip44:118`

Won't this give a chainReference of `cosmoshub-4/slip44`, when really the chainReference should be `cosmoshub-4`? `slip44` would be the asset namespace.